src/components/playback/playbackmanager.js

```diff
@@ -1783,7 +1783,7 @@ class PlaybackManager {
                     MediaTypes: 'Photo,Video',
                     Limit: UNLIMITED_ITEMS
                 }).then(function (result) {
-                    const items = result.Items;
+                    items = result.Items;
```

Comment:
Same here: Overwriting the variable in the outer scope does not appear to be the desired behavior here.